### PR TITLE
Fixed nodePort for dashboard #185

### PR DESCRIPTION
**Pull request by rdsubhas**
* Part of #185, sorry for the temporary radio silence
* Marked as WIP to check the build status, temporarily set nodePort as 30000
* Checked kubernetes dashboard UI, seems more trickier than initially thought, will deal that as a separate PR

### CODE REVIEW

**Comment by rdsubhas** [comment]
I signed it!

**Comment by rdsubhas** [comment]
`addons_test` doesn't seem to be triggered in the PR checks, checking it out...

**Comment by dlorenc** [comment]
Ah yeah, the integration tests are not run in Travis. You can trigger those locally with:
`make integration`

**Comment by dlorenc** [comment]
Or to run just one test:
`go test ./test/integration/ --tags=integration --run=TestAddons`

File: test/integration/addons_test.go [hunk 1]
```diff
@@ -31,7 +31,8 @@ import (
 
 var (
 	addonManagerCmd = []string{"get", "pod", "kube-addon-manager-127.0.0.1", "--namespace=kube-system"}
-	dashboardCmd    = []string{"get", "rc", "kubernetes-dashboard", "--namespace=kube-system"}
+	dashboardRcCmd  = []string{"get", "rc", "kubernetes-dashboard", "--namespace=kube-system"}
+	dashboardSvcCmd = []string{"describe", "svc", "kubernetes-dashboard", "--namespace=kube-system"}
```
Comment (dlorenc) [inline test/integration/addons_test.go]: I pulled in your change to test it out, I think you need to change this to "get", "svc". We parse the output using -o=json, and describe doesn't support that flag.

**Comment by dlorenc** [comment]
Thanks! I patched this in locally and tests look good!

```shell
$ go test -v ./test/integration/ --tags=integration --run=TestDashboard
=== RUN   TestDashboard
--- PASS: TestDashboard (93.17s)
PASS
ok      _/Users/dlorenc/fakego/src/k8s.io/minikube/test/integration     93.247s
```

**Comment by rdsubhas** [comment]
Thanks!

**Pull request merged**